api/provisioner/provisioner.go

```diff
@@ -271,6 +271,7 @@ func (st *State) HostChangesForContainer(containerTag names.MachineTag) ([]netwo
 	for i, bridgeInfo := range newBridges {
 		res[i].BridgeName = bridgeInfo.BridgeName
 		res[i].DeviceName = bridgeInfo.HostDeviceName
+		res[i].BondReconfigureDelay = bridgeInfo.BondReconfigureDelay
```

Comment:
Shouldn't we only be doing this if we know that the underlying device is a bond?